test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
+link_directories(/src/LPM/src /src/LPM/src/libfuzzer /src/LPM/external.protobuf/lib)
```

Comment:
Absolute directories? Did we have that in the other case?

Reply:
No, this is a new requirement introduced by additional external dependency (libProtobufMutator)